lib/latest-semver-tag.js

```diff
@@ -0,0 +1,15 @@
+const gitSemverTags = require('git-semver-tags')
```

Comment:
Could this file be replaced by just using [`git-latest-semver-tag`](https://www.npmjs.com/package/git-latest-semver-tag)?

@stevemao could we also move `git-latest-semver-tag` into the `conventional-changelog` org? (since I seem to be stealing all your other packages recently :grin: )

Reply:
I think I'd honestly rather add getting the latest tag as a feature to `git-semver-tags`, before we pull another repo into the mono-repo. It doesn't feel like there's enough of a functional difference between `git-semver-tags` and `git-latest-semver-tag` to warrant maintaining the additional library.

https://github.com/conventional-changelog/conventional-changelog/issues/388